src/motion/features/layout/Animate.tsx

```diff
@@ -62,7 +62,12 @@ class Animate extends React.Component<AnimateProps> {
             ...config
         }: SharedLayoutAnimationConfig = {}
     ) => {
-        const { visualElement, layout } = this.props
+        const { visualElement, layout, _shouldAnimateLayout } = this.props
+
+        /**
+         * Early return if we've been instructed not to animate this render.
+         */
+        if (_shouldAnimateLayout === false) return this.safeToRemove()
```

Comment:
Is this the only place `_shouldAnimateLayout` is required in the refactored version of magic? Previously there were optimisations to use the lead/follow's `_shouldAnimate` for stack that enabled the optimization to work in Framer. Do we no longer need to support that?

Reply:
Ah qui let me take a look

Reply:
Yeah you were right, just reimplemented